src/test/java/org/jabref/logic/citationstyle/CitationStyleGeneratorTest.java

```diff
@@ -58,6 +60,23 @@ void aPACitation() {
         assertEquals(expected, citation);
     }
 
+    /**
+     * The below test, as of now, fails due to citeproc-java returning an empty citation.
+     * Alphanumeric citations are thus, currently manually generated by formatting (see {@link org.jabref.logic.openoffice.oocsltext.CSLFormatUtils#generateAlphanumericCitation(List, BibDatabaseContext) generateAlphaNumericCitation}).
+     */
+    @Test
+    @Disabled
```

Comment:
Disabled or enabled?

Reply:
Add the comment to @Disabled

Reply:
@ThiloteE it should be disabled because this test fails currently (depends on external).

Reply:
@Siedlerchr The javadoc already conveys that it doesn't work (implying @disabled). Should I explicitly add another comment beside the annotation?

Reply:
I would prefer a short text inside the annotation - that's what it's for. 
e.g. Disabled("Alphanumeric citations are currently not supported by citeproc java")

Reply:
Added (https://github.com/JabRef/jabref/pull/11636/commits/2a611deffd61122e720c8f0344b95c7574be8bb8).
Went with `@Disabled("Till alphanumeric citations are supported by citeproc-java")`